indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -237,6 +216,66 @@
         });
     }
 
+    function setupAjaxForms() {
+        function _getOptions($elem, extra) {
+            return $.extend({
+                context: $elem,
+                formContainer: $elem.data('form-container'),
+                confirmCloseUnsaved: $elem.data('confirm-close-unsaved') !== undefined,
+                update: {
+                    element: $elem.data('update'),
+                    replace: $elem.data('replace-update') !== undefined,
+                    highlight: $elem.data('highlight-update') !== undefined
+                }
+            }, extra);
+        }
+
+        function _handleClick(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            if ($this.hasClass('disabled')) {
+                return;
+            }
+
+            inlineAjaxForm(_getOptions($this, {
+                load: {
+                    url: $this.data('href')
+                }
+            }));
+
+            function toggleDisabled(disabled) {
+                if ($this.data('hide-trigger') !== undefined) {
+                    $this.toggle(!disabled);
+                }
+                if ($this.is('a')) {
```

Comment:
I think `is` doesn't play well with IE. Please make sure it works fine, I had a similar problem with a widget I was working on.